skll/learner.py

```diff
@@ -1109,18 +1132,27 @@ def model_params(self):
         # "0 vs 2", ... "0 vs n", "1 vs 2", "1 vs 3", "1 vs n", ... "n-1 vs n".
         elif isinstance(self._model, SVC) and self._model.kernel == 'linear':
             intercept = {}
+            if isinstance(self.feat_vectorizer, FeatureHasher):
+                self.logger.warning("No feature names are available since this model was trained on hashed features.")
             for i, class_pair in enumerate(combinations(range(len(self.label_list)), 2)):
                 coef = self.model.coef_[i]
                 coef = coef.toarray()
                 coef = self.feat_selector.inverse_transform(coef)[0]
                 class1 = self.label_list[class_pair[0]]
                 class2 = self.label_list[class_pair[1]]
-                for feat, idx in iteritems(self.feat_vectorizer.vocabulary_):
-                    if coef[idx]:
-                        res['{}-vs-{}\t{}'.format(class1, class2, feat)] = coef[idx]
-
+                if isinstance(self.feat_vectorizer, FeatureHasher):
```

Comment:
Not a big deal, but we're repeating this code a lot, and the only thing that's really changing is the actual feature name strings. Is there any way to goup this into a function, or is that too annoying?

Reply:
Good call! I'll try and see if some refactoring is possible here.